### ISSUE TITLE
I get 502 in production after I log into manager

### ISSUE THREAD

**Issue by traustitj** [PiranhaCMS/piranha.core#1268]
I have a production site up and running. I put the website to mysql, followed the introduction on https://piranhacms.org/docs/architecture/databases/mysql

On my local (windows) machine, I can log into the manager, but on production (linux) I can not.

I have not touched the user table or anything like that.

But after I log in (admin/password) I get 502 Bad Gateway

**Comment by traustitj** [comment]
I found out why. I am using nginx and I had to put this inside the http part in /etc/nginx/nginx.conf

  proxy_buffer_size   128k;
  proxy_buffers   4 256k;
  proxy_busy_buffers_size   256k;

**Comment by asadsahi** [comment]
Can't explain, but this magic works for me too 😆

**Comment by PoisnFang** [comment]
Why does this work?

**Comment by mikaellindemann** [comment]
From what I have read on the chat and other issues, it is because the accumulated headers after logging in exceeds the default buffer size of nginx's proxy

**Comment by surentharp** [comment]
I also faced 502 error after trying authenticating to the piranha manager. The nginx error log showed "upstream sent too big header while reading response header from upstream".

I was following the [Microsoft documentation](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/linux-nginx?view=aspnetcore-6.0) for setting up the proxy using a separate file through etc/nginx/proxy.conf

In proxy.conf, I included the proxy_buffer_size and proxy_busy_buffers_size, then increased proxy_buffers. After this modification, i am not getting 502 error after piranha manager authentication. It is working fine.
